_posts/2022-02-08-opensearch-java-runtime.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "Using different Java runtimes with OpenSearch"
+authors:
+  - dblock
+  - smortex
+  - reta
+  - ryanbogan
+date: 2022-02-08
+categories:
+  - technical-post
+twittercard:
+  description: "OpenSearch ships with a bundled Java Development Kit (JDK) that has recently been updated to version 17 (LTS). In this blog post we'll explain this change, and describe new features that make swapping the JDK easier."
+excerpt: ""
+
+---
+At the time of the fork OpenSearch inherited bundling OpenJDK 15, and we have made 8 releases with AdoptOpenJDK 15.0.1+9 as the default runtime, replaced with Adoptium (Temurin) 17.0.2+8 in OpenSearch 1.3.0, which is a Long-Term Support (LTS) release. LTS releases focus on stability, therefore you can expect future versions of OpenSearch to always default to bundling an LTS JDK.
+
+| Version                                                         | Bundled JDK (Linux)         |
+|:---------------------------------------------------------------:|:---------------------------:|
+| [1.0.0](https://opensearch.org/versions/opensearch-1-0-0.html)  | AdoptOpenJDK 15.0.1+9       |
```

Comment:
It might be cleaner to just have `1.0.0 - 1.2.4` as a single row. Or have a `1.1` row, `1.2` row, and `1.3` row. I scanned this table to see if there were any differences, but there were not. Consolidating it should make reading through this post a little easier.